src/cephadm/cephadm

```diff
@@ -112,6 +112,200 @@ class Ceph(object):
 
 ##################################
 
+class Firewalld(object):
+    def __init__(self):
+        # type () -> None
+        self.disabled = False
+        self.available = False
+        self.check()
+
+    def check(self):
+        # type () -> None
+        self.cmd = find_executable('firewall-cmd')
+        if not self.cmd:
+            logger.debug('firewalld does not appear to be present')
+            return
+        (enabled, state, _) = check_unit('firewalld.service')
+        if not enabled:
+            logger.debug('firewalld.service is not enabled')
+            return
+        if state != "running":
+            logger.debug('firewalld.service is not running')
+            return
+
+        logger.info("firewalld ready")
+        self.available = True
+
+    def enable_service_for(self, daemon_type):
+        # type (str) -> None
+        if self.disabled:
+            return
+        if not self.available:
+            logger.debug('Not possible to enable service <%ss>. firewalld.service is not available' % daemon_type)
+            return
+
+        fw_services = []
+        if daemon_type == 'mon':
+            fw_services.append('ceph-mon')
+        elif daemon_type in ['mgr', 'mds', 'osd']:
+            fw_services.append('ceph')
+        elif daemon_type == NFSGanesha.daemon_type:
+            fw_services.append('nfs')
+
+        for svc in fw_services:
```

Comment:
`enable_service_for` is called with a `type->str` argument, hence `len(fw_services)` should never be `> 1`,  right?

What's the point of using a `list` here?

Reply:
You're completely right .. ( this things happens when moving code) thx!